File: ios/UGames/Network/DTOs/GameDTO.swift
import Foundation

struct GameDTO: Decodable {
    let appID: Int64
    let title: String
    let rating: Double?
    let ratingCount: Int?
    let media: GameMediaDTO?
    let categoriesNames: [String]?
    let developer: NamedDTO?
    let features: GameFeaturesDTO?
}

File: android/app/src/main/kotlin/games/yandex/wrap/network/dtos/GameDto.kt
package games.yandex.wrap.network.dtos

import kotlinx.serialization.Serializable

@Serializable
data class GameDto(
    val appID: Long,
    val title: String,
    val rating: Float = 0f,
    val ratingCount: Int = 0,
    val media: GameMediaDto? = null,
    val categoriesNames: List<String> = emptyList(),
    val developer: NamedDto? = null,
    val features: GameFeaturesDto? = null,
)
